### PR TITLE
Update 'BFS health finance GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22bfs-health-finance%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22credit%20agency%22%5D%2C%22name%22%3A%22BFS%20health%20finance%20GmbH%22%2C%22address%22%3A%22Carl-Bertelsmann-Stra%C3%9Fe%20270%5Cn33311%20G%C3%BCtersloh%5CnDeutschland%22%2C%22phone%22%3A%22%2B49%20800%204473254%22%2C%22fax%22%3A%22%2B49%20231%20945362888%22%2C%22email%22%3A%22datenschutz%40meinebfs.de%22%2C%22web%22%3A%22https%3A%2F%2Fmeinebfs.de%2F%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fmeinebfs.de%2Fdatenschutz%2F%22%2C%22https%3A%2F%2Fmeinebfs.de%2Fimpressum%2F%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1061%22%5D%2C%22needs-id-document%22%3Atrue%2C%22suggested-transport-medium%22%3A%22email%22%2C%22comments%22%3A%5B%22Die%20BFS%20health%20finance%20GmbH%20selbst%20sitzt%3A%20H%C3%BClshof%2024%2C%2044369%20Dortmund%2C%20Deutschland.%5Cn%5CnEntweder%20geschw%C3%A4rzte%20Ausweiskopie%20oder%20BFS-Nummer%20wird%20gefordert.%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/bfs-health-finance.json [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
     "categories": [
-        "credit agency"
+        "credit agency",
+        "health"
     ],
     "name": "BFS health finance GmbH",
-    "address": "Carl-Bertelsmann-Straße 270\n33311 Gütersloh\nDeutschland",
-    "phone": "+49 800 447 32 54",
-    "fax": "+49 231 945 362 888",
-    "email": "datenschutz@meinebfs.de",
```
Comment (mal-tee) [inline companies/bfs-health-finance.json:10]: I couldn't find a source for this. :/

File: companies/bfs-health-finance.json [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
+        {
+            "desc": "BFS-Nummer",
+            "type": "input",
+            "optional": true
+        }
     ],
-    "relevant-countries": [
-        "de"
+    "comments": [
+        "Die BFS health finance GmbH selbst sitzt: Hülshof 24, 44369 Dortmund, Deutschland.\n\nEntweder geschwärzte Ausweiskopie oder BFS-Nummer wird gefordert."
```
Comment (mal-tee) [inline companies/bfs-health-finance.json:29]: Should we keep this here or move it to a backend-powered comment?

File: companies/bfs-health-finance.json [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
+        "de"
+    ],
     "categories": [
-        "credit agency"
+        "credit agency",
+        "health"
     ],
     "name": "BFS health finance GmbH",
-    "address": "Carl-Bertelsmann-Straße 270\n33311 Gütersloh\nDeutschland",
-    "phone": "+49 800 447 32 54",
```
Comment (mal-tee) [inline companies/bfs-health-finance.json:8]: this number is in the impressum, while the other is on the privacy page

**Review by mal-tee — COMMENTED** [review]
Just some small notes; I would appreciate your input, Benni :D

File: companies/bfs-health-finance.json (hunk 1 above)
Comment (baltpeter) [reply companies/bfs-health-finance.json:10]: They do still have a press release mentioning the email on their site (https://meinebfs.de/dsgvo-die-datenschutzgrundverordnung/) but it does seem like there was some restructuring and they are not using that email anymore, unfortunately.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**